src/components/BoxSelection/Option.st.css

```diff
@@ -0,0 +1,211 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback, font, applyOpacity;
+}
+
+:import {
+    -st-from: "../../common/palette.st.css";
+    -st-named: errorColor, focusRing;
+}
+
+/*Defaults*/
+:vars {
+    DefaultSelectedBoxColor: color-8;
+    DefaultBoxFillColor: color-1;
+    DefaultBoxFillOpacity: 1;
+    DefaultBorderColor: color-5;
+    DefaultBoxBorderWidth: 1px;
+    DefaultBoxBorderRadius: 0px;
+    DefaultBoxPadding: 16px;
+    DisabledColor: color-3;
+    DefaultTextColor: color-5;
+    DefaultDisabledTextColor: color-3;
+}
+
+/*Overrides*/
+:vars {
+    /*
+        The BG color of the box
+        @default color-1
+    */
+    BoxFillColor: --overridable;
+    /*
+        The border color of the box (unavailable line as well)
+        @default color-5
+    */
+    BoxBorderColor: --overridable;
+    /*
+        The BG opacity of the box
+        @default 1 (100%)
+    */
+    BoxFillOpacity: --overridable;
+    /*
+        The border width of the box (unavailable line as well)
+        @default 1px 
+    */
+    BoxBorderWidth: --overridable;
+    /*
+        The border radius for each box
+        @default 0px
+    */
+    BoxBorderRadius: --overridable;
+     /*
+        The padding of the box. Predefined by size prop
+        @default 16px
+    */
+    BoxPadding: --overridable;
+    /*
+        The highlight color of a selected option
+        @default color-8
+    */
+    SelectedBoxColor: --overridable;
+    /*
+        Text color of the box
+        @default color-5
+    */
+    TextColor: --overridable;
+    /*
+        Text color of the box when disabled or Unavailable
+        @default color-3
+    */
+    DisabledTextColor: --overridable;
```

Comment:
I'm not sure Johanna meant we need to open a variable for the disabled color, but it's ok to leave it now.
I'm checking with her

Reply:
so I checked with @johannaasseraf, and she said don't need this variable.
Her comment was about having the correct default color (color-3), but it doesn't need to be overridable by a variable

Reply:
removed